README.md

```
@@ -1 +1,49 @@
+
+## Repo structure
+
+The folder structure in this repository is intended to behave much like a simple
+hierarchical navigation. Documents higher in the tree are more generally
+applicable, while those further down are more specialized and may even override
+or contradict more general advice, in more specific contexts represented by their
+structural specificity.
+
+For example:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)